source/shared/core_conn.cpp

```diff
@@ -233,13 +268,58 @@ sqlsrv_conn* core_sqlsrv_connect( _In_ sqlsrv_context& henv_cp, _In_ sqlsrv_cont
     return return_conn;
 }
 
+// core_search_odbc_driver
+// This method is meant to be used in a non-Windows environment, 
+// searching for a particular ODBC driver name in the odbcinst.ini file
+// Parameters:
+// driver_version   - a valid value in enum DRIVER_VERSION
+// Return           - a boolean flag that indicates if the specified driver version is found or not
+
+bool core_search_odbc_driver( _In_ std::size_t driver_version )
```

Comment:
can we rename this to something that indicates it's a unix_search_odbc_driver or something?